swig/include/python/gdal_python.i

```diff
@@ -2809,6 +2809,9 @@ def Warp(destNameOrDestDS, srcDSOrSrcDSTab, **kwargs):
     else:
         srcDSTab = [srcDSOrSrcDSTab]
 
+    if destNameOrDestDS is None:
```

Comment:
I assume a better fix would be to add
```
%apply Pointer NONNULL {const char* dest};
```
before  the GDALDatasetShadow* wrapper_GDALWarpDestName( ) declaration in gdal.i
and
```
%clear const char* dest;
```
afterwards

Reply:
Done (actually to wrapper_GDALWarpDestDS)